### PR TITLE
Crato applying for Grant

### PR DESCRIPTION
**Pull request by halcyon-zihao**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/crustio/Crust-Grants-Program/blob/main/applications/application_template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A Ethereum (USDT/CRU) address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md).
- [x] I accept the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md) and submit the PR.
- [x] The total funding amount of the project is below USDT $30k.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once we've successfully [delivered](https://github.com/crustio/Crust-Grant-Milestone-Delivery) the first milestone.

### CODE REVIEW

File: applications/crato.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+* provide a test suite, comprising unit and integration tests, along with a guide on how to set up and run them.
+* commit to providing Dockerfiles for the delivery of your project. 
+* indicate milestone duration as well as number of full-time employees working on each milestone, and include the approximate number of days along with the cost per day.
+* _Deliverables 0a-0d are mandatory_ and shall not be removed, unless you explicitly specify a reason within the PR's `Additional Notes` section (e.g. Milestone X is research oriented and as such there is no code to test).
+
+> If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+* **Total Estimated Duration:** 3 months
+* **Full-Time Equivalent (FTE):**  3 FTE
```
[the remark this answers is not in the document]
Comment (halcyon-zihao) [reply applications/crato.md:86]: Yes, we have 2.

**Review by halcyon-zihao — COMMENTED** [review]
I have fixed the information as Lei Lu requested.

File: applications/crato.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* commit to providing Dockerfiles for the delivery of your project. 
+* indicate milestone duration as well as number of full-time employees working on each milestone, and include the approximate number of days along with the cost per day.
+* _Deliverables 0a-0d are mandatory_ and shall not be removed, unless you explicitly specify a reason within the PR's `Additional Notes` section (e.g. Milestone X is research oriented and as such there is no code to test).
+
+> If any of your deliverables is based on somebody else's work, make sure you work and publish _under the terms of the license_ of the respective project and that you **highlight this fact in your milestone documentation** and in the source code if applicable! **Teams that submit others' work without attributing it will be immediately terminated.**
+
+### Overview
+* **Total Estimated Duration:** 3-6 months
+* **Full-Time Equivalent (FTE):**  6 FTE
+* **Total Costs:** 100,000 USD
```
Comment (yyd106) [inline applications/crato.md:88]: Please give a value lower than 30000 USD

File: applications/crato.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+Please provide the following:
+  * A project name: Crato
+  * A brief description of your project: Sophisticated Pin Service Built On IPFS
+  * An indication of how your project relates to / integrates into Crust: Crato seriously takes its role as eco-partner of Crust to
+provide decentralized meta data management service with easy
+IPFS API and toolkit.
+  * An indication of why your team is interested in creating this project: We have always been interested in field of Storage, by the power of IPFS and Crust blockchain, we have chance to creat a brand new storage service.
+
+### Project Details
```
Comment (yyd106) [inline applications/crato.md:22]: Missing key information, please fill in it completely

File: applications/crato.md (hunk 2 above)
Comment (halcyon-zihao) [reply applications/crato.md]: Already updated at your request.

File: applications/crato.md (hunk 3 above)
Comment (halcyon-zihao) [reply applications/crato.md:22]: Already updated at your request.

**Pull request closed without merging**